helm/index.go

```diff
@@ -0,0 +1,50 @@
+package helm
+
+import (
+	"sort"
+	"time"
+
+	"github.com/Masterminds/semver/v3"
+)
+
+// IndexFile represents the index file in a chart repository.
+type IndexFile struct {
+	ServerInfo  map[string]interface{}   `json:"serverInfo,omitempty"`
+	APIVersion  string                   `json:"apiVersion"`
+	Generated   time.Time                `json:"generated"`
+	Entries     map[string]ChartVersions `json:"entries"`
+	PublicKeys  []string                 `json:"publicKeys,omitempty"`
+	Annotations map[string]string        `json:"annotations,omitempty"`
+}
+
+func (i IndexFile) SortEntries() {
+	for _, versions := range i.Entries {
```

Comment:
Q&A: so versions is reference and Sort sorts it in place?

Reply:
Yes